.github/workflows/dockerimage-latest.yml

```diff
@@ -1,59 +1,60 @@
-name: Dockerimage latest
+name: Dockerimage Builder
 
 on:
   push:
     branches: 
       - master
+      - refactor
+  workflow_dispatch:
 
 jobs:
-  CPU:
+  latest:
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v2
-      with:
-        ref: 'master'
     - name: Build the Docker image
       env:
         HUB_NAME: ${{ secrets.DOCKER_NAME }}
         HUB_KEY: ${{ secrets.DOCKER_TOKEN }}
-      run: |  
+      run: |
         docker login -u $HUB_NAME -p $HUB_KEY
-        docker build . --pull --no-cache --file Dockerfile --tag zocker160/blender-crowdrender:latest
-        docker tag zocker160/blender-crowdrender:latest zocker160/blender-crowdrender:bl_2.93
-        docker push zocker160/blender-crowdrender:latest
-        docker push zocker160/blender-crowdrender:bl_2.93
+        
+        docker build . --pull --no-cache --file Dockerfile --tag crowdrender/blender-crowdrender:latest
+        
+        docker tag crowdrender/blender-crowdrender:latest crowdrender/blender-crowdrender:bl_3.2
+        
+        docker push crowdrender/blender-crowdrender:latest
+        docker push crowdrender/blender-crowdrender:bl_3.2
 
-  nvidia10:
+  LTSone:
```

Comment:
Looks like one and two are used to differentiate 3.2 and 2.79, do we think an alternative naming scheme could be more descriptive and continue to be intuitive with future LTS versions?

Reply:
Yes, in fact there will be another PR to address this. Currently working on a [branch](https://github.com/crowdrender/cr-docker/tree/fix-machine-uuid-overriding) where we can specify all the different versions via env vars and then tag them appropriately. So the tags will be no longer hardcoded, but specified by env vars.